app/common/locales/en.json

```diff
@@ -1080,8 +1080,26 @@
         "pending": "<strong>Status:</strong> Pending",
         "failed": "<strong>Status:</strong> Failed",
         "queued": "<strong>Status:</strong> In Progress",
-        "export": "Export",
-        "export_history": "Export History"
+        "tags_attributes": "Assign tags and attributes",
+        "hxl_apikey_alert_1": "It looks like you haven't set up your Humanitarian Data Exchange (HDX) API. Without this set up, you can't tag data and upload to a Humanitarian Data Exchange (HDX) account.",
+        "hxl_configure": "Configure HDX API",
+        "hxl_apikey_alert_2": "We strongly advide you set this up now in",
+        "hxl_apikey_alert_3": "before continuing as you won't be able to upload your data to HDX",
+        "hxl_title": "Assign your HXL tags and attributes, select fields and export or upload",
+        "hxl_desc": "Select specific fields from a survey, assign a tag per field and then choose to assign multiple attributes. Preview your tag and attributes in tag preview.",
+        "hxl_instructions_1": "You can then choose to export your tagged fields to a CSV file or upload to a",
```

Comment:
@Angamanga the translations seem to be broken, can you check? (ie data_export.export instead of the text)

Reply:
fixed!